sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/WebAppBaseImpl.java

```diff
@@ -1835,6 +1836,33 @@ public FluentImplT withoutIpAddressRangeAccess(String ipAddressCidr) {
         return (FluentImplT) this;
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public FluentImplT enablePublicNetworkAccess() {
+        if (Objects.isNull(this.siteConfig)) {
+            this.siteConfig = new SiteConfigResourceInner();
+        }
+        this.siteConfig.withPublicNetworkAccess("Enabled");
+        return (FluentImplT) this;
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public FluentImplT disablePublicNetworkAccess() {
+        if (Objects.isNull(this.siteConfig)) {
+            this.siteConfig = new SiteConfigResourceInner();
+        }
+        this.siteConfig.withPublicNetworkAccess("Disabled");
+        return (FluentImplT) this;
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public PublicNetworkAccess publicNetworkAccess() {
+        return Objects.isNull(innerModel().publicNetworkAccess())
+            && innerModel().publicNetworkAccess().trim().isEmpty() ? null : PublicNetworkAccess.fromString(innerModel().publicNetworkAccess());
```

Comment:
No need to trim or check empty, just do `PublicNetworkAccess.fromString`, if not null.